towhee/pipelines/image_embedding_pipeline.py

```diff
@@ -99,6 +108,50 @@ def image_embedding_pipeline(model: Union[str, List[str]] = None,
                              ensemble: str = None,
                              name: str = None,
                              version: str = None):
+    """Create a pipeline for image embedding tasks.
+
+    An image embedding pipeline converts input images into feature vectors (embedding),
+    which can be adapted to various vision tasks,
+    such as image retrieval, image classifications, etc.
+
+    There are tow ways to instantiate an image embedding pipeline:
+
+    1 - If `model` is passed to `image_embedding_pipeline`,
+    a new pipeline will be generated for evaluation and benchmarking.
+
+    ```python
+    >>> pipe = image_embedding_pipeline(model='resnet101')
+    ```
+
+    The pipeline can be saved to file if the evaluation results seems good,
+    and you want to reuse this pipeline.
+    ```python
+    >>> pipe.save('my_image_embedding_pipeline', 'my_pipelines')
+    ```
+
+    You can also publish this pipeline to towhee hub to share it with the community.
+    ```shell
+    $ cd ${WORK_DIR}/my_pipelines/my_image_embedding_pipeline
+    $ towhee publish # see publish guild from the terminal
```

Comment:
how about 
towhee publish path/to/pipeline

Reply:
we should consider adding a `publish` method, i mean, a python interface for publishing a pipeline

Reply:
> how about
> towhee publish path/to/pipeline

`towhee publish` only print user guide for linking local repo with towhee hub. The user needs git to push his code to the hub.

Reply:
> we should consider adding a `publish` method, i mean, a python interface for publishing a pipeline

I agree, we can update source code in the repo directly with restful API.